website/docs/getting-started/docker-compose.md

```diff
@@ -0,0 +1,192 @@
+# Docker Compose
+
+The easiest way to run the ORT Server for testing is to use [Docker Compose](https://docs.docker.com/compose/).
+For a proper deployment to Kubernetes, the project will later provide a Helm chart.
+
+:::warning
+Do not use the Docker Compose setup in production as it uses multiple insecure defaults, like providing Keycloak without TLS.
+:::
+
+To use Docker Composer, you need to have [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed.
+Also, you have to clone the ORT Server repository:
+
+```shell
+git clone https://github.com/eclipse-apoapsis/ort-server.git
+```
+
+## Run with Published Images
+
+To start the ORT Server with the required third-party services, run:
+
+```shell
+docker compose up
+```
+
+This will use the ORT Server images [published on GitHub](https://github.com/orgs/eclipse-apoapsis/packages?ecosystem=container).
+By default, the `main` tag is used.
+Those images are built from the `main` branch of the repository.
+To use a different tag, you can set the `ORT_SERVER_IMAGE_TAG` environment variable, for example:
+
+```shell
+ORT_SERVER_IMAGE_TAG=0.1.0-SNAPSHOT-001.sha.aa4d3fa docker compose up
+```
+
+By default, the ORT Server API is exposed on port 8080.
+If this port is already in use, it can be changed using the `ORT_SERVER_CORE_PORT` environment variable:
+
+```shell
+ORT_SERVER_CORE_PORT=8090 docker compose up
+```
+
+When using a different port, please make sure that it is not used by [another service](#accessing-the-services).
+
+### Update the images
```

Comment:
Nit: You seem to be inconsistently using title-case for headers in this document; e.g. here, the "Image" noun should probably be capitalized.

Reply:
I read that several style guides recommend using title case only for the first two levels. However,  I did some research again and this seems not to be up-to-date. For example, Microsoft style guide recommends to only use sentence case (https://learn.microsoft.com/en-us/style-guide/scannable-content/headings) and the APA switched from using title case only for the first two levels to using it for all headings (https://academicguides.waldenu.edu/writingcenter/apa/other/headinglevels). So for simplicity I'll go with the APA recommendation.

Reply:
> So for simplicity I'll go with the APA recommendation.

Fine with me as long as it's consistently applied.